Mobile-Workshop/Mobile-Workshop/ProductDetail/ProductDetailView.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  ProductDetailView.swift
+//  Mobile-Workshop
+//
+//  Created by iOS Dev on 14.05.23.
+//
+import SwiftUI
+import Foundation
+import SwiftUI
+
+struct ProductDetailView: View {
+    @State public var text: String = ""
+    
+    var body: some View {
+        VStack(spacing: 0) {
```

Comment:
I believe the default spacing is `nil`, so there shouldn't be any. Is there a particular reason for the explicit 0?

Reply:
Incorrect. The default spacing is somewhere around 8, it should always be set to a 0 manually.